coraza.conf-recommended

```diff
@@ -115,15 +118,16 @@ SecDataDir /tmp/
 #
 #SecUploadDir /opt/coraza/var/upload/
 
-# By default, only keep the files that were determined to be unusual
-# in some way (by an external inspection script). For this to work you
-# will also need at least one file inspection rule.
+# If On, the WAF will store the uploaded files in the SecUploadDir
+# directory.
+# Note: SecUploadKeepFiles is currently NOT supported by Coraza
 #
-#SecUploadKeepFiles RelevantOnly
+#SecUploadKeepFiles Off
 
 # Uploaded files are by default created with permissions that do not allow
 # any other user to access them. You may need to relax that if you want to
 # interface Coraza to an external program (e.g., an anti-virus).
+# Note: SecUploadFileMode is currently NOT supported by Coraza
```

Comment:
We are calling  [`os.CreateTemp(storagePath, "crzmp*")` ](https://github.com/corazawaf/coraza/blob/e9f4e803aa184a0e8fd2057e565eb0bd3226da1c/internal/bodyprocessors/multipart.go#L63-L67), which is coming with hardcoded `0600`. So we are reading this value, not effectively not using it.